extensions/amp-story-shopping/0.1/test/test-amp-story-shopping-tag.js

```diff
@@ -28,15 +43,39 @@ describes.realWin(
         'amp-story-shopping-tag',
         {'layout': 'container'}
       );
+
       pageEl.appendChild(element);
       win.document.body.appendChild(pageEl);
 
       shoppingTag = await element.getImpl();
     }
 
-    it('should build shopping tag component', () => {
+    async function shoppingStateHelper() {
+      const shoppingState = {
+        'sunglasses': {'product-title': 'Spectacular Spectacles'},
+      };
+      storeService.dispatch(Action.ADD_SHOPPING_DATA, shoppingState);
+      await shoppingTag.buildCallback();
+      await shoppingTag.layoutCallback();
+      await nextTick();
```

Comment:
If this is meant to wait for the `mutateElement` to run and update the textContent, please make mutateElement synchronous instead, e.g. `env.sandbox.stub(shoppingTag, 'mutateElement').callsFake((fn) => fn());`